packages/rx_bloc_cli/mason_templates/bricks/lib_auth_matrix/__brick__/lib/feature_auth_matrix/views/auth_matrix_page.dart

```diff
@@ -19,72 +21,80 @@ class AuthMatrixPage extends StatelessWidget {
         appBar: AppBar(
           title: Text(context.l10n.featureAuthMatrix.authMatrixAppBarTitle),
         ),
-        body: Column(
-          mainAxisAlignment: MainAxisAlignment.center,
-          children: <Widget>[
-            RxBlocListener<AuthMatrixBlocType, void>(
-              state: (bloc) => bloc.states.response,
-              listener: (context, _) => _onResponse(context),
-            ),
-            AppErrorModalWidget<AuthMatrixBlocType>(
-              errorState: (bloc) => bloc.states.errors.translate(context),
-            ),
-            Padding(
-              padding: EdgeInsets.all(context.designSystem.spacing.xl),
-              child: TextFieldDialog<String>(
+        body: Padding(
+          padding: EdgeInsets.all(context.designSystem.spacing.l),
+          child: Column(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: [
+              AppErrorModalWidget<AuthMatrixEditAddressBlocType>(
+                errorState: (bloc) => bloc.states.error,
+              ),
+              RxBlocListener<AuthMatrixEditAddressBlocType, AuthMatrixAction>(
+                state: (bloc) => bloc.states.onAuthMatrixCompleted,
+                listener: _showAuthMatrixComplete,
+              ),
+              Text(
+                'The address component below demonstrates the 2FA (PIN and OTP) feature.',
```

Comment:
I guess we don't want to add these strings to `l10n` since they're just here for demo?

Reply:
Good catch. The strings should be translated